msal/src/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -82,30 +97,44 @@ public abstract partial class ClientApplicationBase
         }
 
         /// <summary>
-        /// Identifier of the component consuming MSAL and it is intended for libraries/SDKs that consume MSAL. This will allow for disambiguation between MSAL usage by the app vs MSAL usage by component libraries.
+        /// Identifier of the component (libraries/SDK) consuming MSAL.NET. 
+        /// This will allow for disambiguation between MSAL usage by the app vs MSAL usage by component libraries.
```

Comment:
I suppose we should ask developers to use this as that will allow us to provide better error messages, etc. ... Perhaps we should add more information on why developers would want to set this. and an aka.ms link to a more detailed description of PublicClientApplication and/or ConfidentialClientApplication in the wiki?

Reply:
Done in the summary section of the app